recipes/proposal/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "7.0.2":
+    url: "https://github.com/tudo-astroparticlephysics/PROPOSAL/archive/refs/heads/7.0.2.tar.gz"
+    sha256: "922b33eb9f4ca0ebf7a4b8a85b5d2642733cb102c6cf511ce72d99ce0a08771b"
```

Comment:
Surprisingly it works, but I don't see this version in upstream repo: https://github.com/tudo-astroparticlephysics/PROPOSAL/releases